### ISSUE TITLE
Simplify routing

### ISSUE THREAD

**Issue by Burnett01** [Burnett01/sys-api#2]
I think we should re-design the routing-pattern. 

As of now it looks like:

``` coffeescript
api.get('/heartbeat', (req, res, next) ->
    api.response(req, res, next, "dub")
)
```

What it should look like instead:

``` coffeescript
api.get('/heartbeat', (req, res, next) ->
    api.response("dub")
)
```

This would follow DRY.

@Cloud2Box/developers

**Comment by Burnett01** [comment]
I created a new branch where we can work on this:

https://github.com/Cloud2Box/sys-api/tree/new-routing-pattern

**Comment by Burnett01** [comment]
Resolved with #4
